src/components/TranslationBanner.tsx

```diff
@@ -105,9 +111,11 @@ const TranslationBanner: React.FC<IProps> = ({
                   mt={{ base: 2, sm: 0 }}
                   borderColor="#333333"
                   color="#333333"
-                  language={DEFAULT_LOCALE}
+                  lang={locale}
```

Comment:
This should be the `DEFAULT_LOCALE` since this button is to navigate to the English version of the page